### PR TITLE
Feat 110 part 3 : Stylize demos :art: 

### PR DESCRIPTION
**Pull request by glepretre**
image

Mockups: https://github.com/Kozea/formol/pull/114#issuecomment-2150306294

### CODE REVIEW

File: assets/formol-baseline-logo.svg [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="299" height="103" viewBox="27 27 244 27">
```
Comment (glepretre) [inline assets/formol-baseline-logo.svg:1]: @annabellou @gentooboontoo I've changed the `viewBox` of this svg to crop the bottom margin. 
Feel free if we should not or you have another idea (dedicated no-margin logo) ?

**Comment by glepretre** [comment]
@annabellou : We could add a favicon to the page

idea:

image

File: assets/formol-baseline-logo.svg (hunk 1 above)
Comment (annabellou) [reply assets/formol-baseline-logo.svg:1]: @glepretre , I can give you logo without margins if you want, maybe it's simplier ?

**Comment by annabellou** [comment]
> We could add a favicon to the page

Yes 👍 I'll do the image.

**Comment by annabellou** [comment]
@glepretre : I pushed some CSS changes for the demos page ! I forgot the favicon image, i'll do it in a separated commit.

**Comment by glepretre** [comment]
PR deployed on https://glepretre.github.io/formol2/ for those who want a final check : @gentooboontoo @alexandreNoens @annabellou 

@annabellou I've tried to add a basic routing, but...

one-does-not-simply

**Review by annabellou — APPROVED** [review]

**Comment by annabellou** [comment]
@glepretre well done, I approve this pull request 👏

**Pull request merged**